cg/store/models.py

```diff
@@ -895,7 +889,7 @@ def to_dict(self, links: bool = False, flowcells: bool = False) -> dict:
         if links:
             data["links"] = [link_obj.to_dict(family=True, parents=True) for link_obj in self.links]
         if flowcells:
-            data["flowcells"] = [flowcell_obj.to_dict() for flowcell_obj in self.flowcells]
+            data["flowcells"] = [flowcell_obj.to_dict() for flowcell_obj in self.flow_cells]
```

Comment:
```suggestion
            data["flowcells"] = [flow_cell.to_dict() for flow_cell in self.flow_cells]
```
when you already touch the line :)